config/locales/de_CH.yml

```diff
@@ -4070,6 +4070,8 @@ de_CH:
       shipment:
         cannot_ready: "Versand nicht möglich."
       invalid_taxonomy_id: "Ungültige Kategorie-ID"
+      toggle_api_key_view: "Ansicht für Schlüsselgenerierung anzeigen"
+      show_api_key_view_toggled: "Ansicht für die Schlüsselgenerierung hat sich geändert!"
```

Comment:
Translations happen in Transifex. To avoid merge conflicts, we edit only en.yml in pull requests.

Reply:
Oki, then I will adjust these changes there. :)